.github/workflows/build_docs.yml

```diff
@@ -1,44 +1,48 @@
  name: Build Docs
- 
  on:
    push:
-     # Sequence of patterns matched against refs/tags
-     tags:
-       - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
-   workflow_dispatch:
-     inputs:
-       version:
-         description: Manual Doc Build Reason
-         default: test
-         required: false
+     branches:
+     - master
+     - main
  jobs:
    docs:
-     name: build & push docs
+     name: CI (${{ matrix.os }}-${{ matrix.environment-file }})
      runs-on: ${{ matrix.os }}
-     timeout-minutes: 90
+     continue-on-error: ${{ matrix.experimental }}
+     timeout-minutes: 20
      strategy:
        matrix:
          os: ['ubuntu-latest']
-         environment-file: [ci/310.yaml]
+         environment-file: [.ci/39.yml]
          experimental: [false]
      defaults:
        run:
          shell: bash -l {0}
-     
      steps:
-       - name: checkout repo
-         uses: actions/checkout@v4
-       
-       - name: setup micromamba
-         uses: mamba-org/provision-with-micromamba@main
+       - uses: actions/checkout@v2
+       - uses: actions/cache@v2
+         env:
+           CACHE_NUMBER: 0
          with:
-           environment-file: ${{ matrix.environment-file }}
-           micromamba-version: 'latest'
-       
-       - name: make docs
-         run: cd docs; make html
-       
-       - name: commit docs
+           path: ~/conda_pkgs_dir
+           key: ${{ matrix.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles(matrix.environment-file) }}
+       - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Is there a reason for not using the [`mamba-org/setup-micromamba@v1`](https://github.com/pysal/spaghetti/blob/fd8f98e0dd39f11360aab7c7bcb8e3b39da2f9c7/.github/workflows/build_docs.yml#L33C16-L33C45) action here?

Reply:
no... which package has the most current recipes?

Reply:
That's a good question... seems to be a hodgepodge